### PR TITLE
Add TCM audit log docs

### LINKED ISSUE #3855: [TCM] Audit log
**Product:** Tarantool Cluster Manager
**Since:** TCM 1.0.0, Tarantool 3.0
**Audience/target:** admins
**Root document:** TBD
**SME:** @ filonenko-mikhail 

# Details

Audit Log:
- content (events, format)
- enabling and disabling
- configuration
- viewing/searching

### PR DESCRIPTION
**Pull request by p7nov**
Resolves #3855 

Deployment: https://docs.d.tarantool.io/en/doc/gh-3855-tcm-audit/reference/tooling/tcm/tcm_audit_log/

- What is audit log
- Enabling audit logging
- Configuring: file and syslog
- Viewing audit logs
- Entry structure reference
- Event types reference

### CODE REVIEW

**Review by filonenko-mikhail — APPROVED** [review]

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+.. note::
+
+    |tcm| audit log records only events that happen in |tcm| itself.
+    For information about Tarantool audit logging, see :ref:`Audit module <enterprise_audit_module>`.
+
+Audit logging is disabled in |tcm| by default. To start recording events, you need
+to :ref:`enable <tcm_audit_log_enable>` and :ref:`configure <tcm_audit_log_config>` it.
+
+Audit log stores event details in the JSON format. Each log entry contains the
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
The audit log stores event details in the JSON format. Each log entry contains the
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+~~~~~~~~~~~~~~~~~
+
+If you use a centralized log management system based on `syslog <https://datatracker.ietf.org/doc/rfc5424/>`__,
+you can configure |tcm| to send its audit log to your syslog server.
+
+To send |tcm| audit logs to a syslog server:
+
+1.  Go to **Audit settings** and select the **syslog** protocol.
+2.  Enter the syslog server URI and select the network protocol. Typically,
+    ``syslogd`` listens on port 514 and use the UDP protocol.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
    ``syslogd`` listens on port 514 and uses the UDP protocol.
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+-   **Output**. The syslog server URI. Default: 127.0.0.1:514 (localhost).
+-   **Timeout**. The syslog write timeout in the `ISO 8601 duration format <https://en.wikipedia.org/wiki/ISO_8601#Durations>`__.
+    Default: ``PT2S`` (two seconds).
+-   **Priority**. The syslog severity level. Default: ``info``.
+-   **Facility**. The syslog facility. Default: ``local0``.
+
+Selecting audit events to record
+--------------------------------
+
+When audit log is enabled, |tcm| records all audit events listed in :ref:`Event types <tcm_audit_log_event_types>`.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
When the audit log is enabled, |tcm| records all audit events listed in :ref:`Event types <tcm_audit_log_event_types>`.
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+To remove an event type from a filters list, click the cross icon beside it.
+
+..  _tcm_audit_log_view:
+
+Viewing audit log
+-----------------
+
+If audit log is written to a file, you can view it in |tcm|.
+To view audit log in |tcm|, go to the **Audit log** page. On this page, you can
+view or search for events.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
If the audit log is written to a file, you can view it in |tcm| in the **Audit log** page.
On this page, you can view or search for events.
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+
+To view the details of a logged audit event, click the corresponding line in the
+table.
+
+To search for an event, use the search bar on the top of the page. Note that the
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
To search for an event, use the search bar at the top of the page. Note that the
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    |tcm_full_name| is a part of the `Enterprise Edition <https://www.tarantool.io/compare/>`_.
+
+|tcm_full_name| provides the audit logging functionality for tracking user activity
+and security-related events, such as:
+
+*   Successful and failed login attempts.
+*   Access to clusters, their configurations, data model, and stored data.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]: `data model` -> `data models` or `a data model`

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 7]
```diff
@@ -0,0 +1,335 @@
...
+Audit logging is disabled in |tcm| by default. To start recording events, you need
+to :ref:`enable <tcm_audit_log_enable>` and :ref:`configure <tcm_audit_log_config>` it.
+
+Audit log stores event details in the JSON format. Each log entry contains the
+event type, description, time, impacted objects, and other information that
+may be used for incident investigation. The complete list of fields is provided in
+:ref:`Structure of audit log events <tcm_audit_log_structure>`.
+
+In addition to writing audit logs, |tcm| has a built-in interface for reading and
+searching them. For details, see :ref:`Viewing audit logs <tcm_audit_log_view>`.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]: I'd rephrase this smth like this (without an introductory phrase): `|tcm| also has built-in interface for reading and
searching audit logs.`

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 8]
```diff
@@ -0,0 +1,335 @@
...
+To configure audit logging, go to **Audit settings**.
+
+Writing to a file
+~~~~~~~~~~~~~~~~~
+
+To write |tcm| audit logs to a file:
+
+1.  Go to **Audit settings** and select the **file** protocol.
+2.  Specify the name of the audit log file. The file appears in the |tcm| working directory.
+3.  Configure the log files rotation: maximum file size and age, and the number
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
3.  Configure the log files rotation: the maximum file size and age, and the number
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 9]
```diff
@@ -0,0 +1,335 @@
...
+    of files to store simultaneously.
+4.  (Optional) Enable compression of audit log files.
+
+.. TODO: move to UI reference
+
+Configuration parameters:
+
+-   **Output file name**. The name of the audit log file. Default: ``audit.log``
+-   **Max size (in MB)**. The maximum size of the log file before it gets rotated,in megabytes. Default: 100.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
-   **Max size (in MB)**. The maximum size of the log file before it gets rotated, in megabytes. Default: 100.
```

File: doc/reference/tooling/tcm/tcm_audit_log.rst [hunk 10]
```diff
@@ -0,0 +1,335 @@
...
+-   **Max age (in days)**. The maximum age of audit log files in days. Default: 30.
+-   **Compress**. Compress audit log files into ``gzip`` archives when rotating.
+
+Sending to syslog
+~~~~~~~~~~~~~~~~~
+
+If you use a centralized log management system based on `syslog <https://datatracker.ietf.org/doc/rfc5424/>`__,
+you can configure |tcm| to send its audit log to your syslog server.
+
+To send |tcm| audit logs to a syslog server:
```
Comment (andreyaksenov) [inline doc/reference/tooling/tcm/tcm_audit_log.rst]:
```suggestion
If you use a centralized log management system based on `syslog <https://datatracker.ietf.org/doc/rfc5424/>`__,
you can configure |tcm| to send its audit log to your syslog server:
```

**Review by andreyaksenov — APPROVED** [review]
Several minor fixes/suggestions
